unlock-protocol.com/blog/wordpress-plugin.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Wordpress Plugin
+subTitle: Adding a lock to your WordPress.org website has never been easier!
+authorName: Julien Genestoux
+publishDate: September 11, 2019
+description: Wordpress.org is the most popular Content Management System. Adding a lock to your site let's you monetize the words you write without asking anyone's permission!
+image: /static/images/blog/wordpress-plugin/wordpress.png
+---
+
+The world of publishing is [moving away from ads](https://medium.com/unlock-protocol/the-end-of-the-ad-supported-web-d4d093fb462f) as the single way to monetize content. In the last 18 months, most of the largest and most influential publications in the US and Europe have been moving to what they call "direct consumer revenue". This is a great thing because it changes incentives from trying steal their users' attention (click bait, fake news, endless slideshows... etc) to creating value for their members.
```

Comment:
```suggestion
The world of publishing is [moving away from ads](https://medium.com/unlock-protocol/the-end-of-the-ad-supported-web-d4d093fb462f) as the single way to monetize content. In the last 18 months, most of the largest and most influential publications in the US and Europe have been moving to what they call "direct consumer revenue." This is a great thing because it changes incentives from trying steal their users' attention (click bait, fake news, endless slideshows... etc) to creating value for their members.
```

American English style is to put periods and commas inside the quotation marks, if that's important

Reply:
TIL!